src/build.py

```diff
@@ -1277,11 +1268,13 @@ def TestBare():
       assembler=Executable(os.path.join(INSTALL_BIN, 'wast2wasm')),
       indir=s2wasm_out,
       fails=WAST2WASM_KNOWN_TORTURE_FAILURES)
+  common_attrs = ['bare', 'O0']
```

Comment:
That sounds both pythonic and resilient, nice. In my head I was thinking "let's just remember to always pass sets in", which now that I say out loud is a vastly worse idea. :)